### DISCUSSION TITLE
Discussions is very nice

### DISCUSSION THREAD

**Discussion by oxedions** [community/community#2813] [Discussions]
Dear @github_dev_team,

We are using **discussions** on our main repository, and I think both negative **and** positive feedback should be provided, since all of this imply work under the hood.

Just wanted to simply say that **discussions** is a very nice addon for github.

It is well made, simple to use, and allows centralized exchanges on everything that is not an issue (simple questions of code users, plan future changes, etc). It makes repositories and projects related more "alive".

So many thanks for this feature 😊

**Comment by becca** [comment]
🤗 thank you so much @oxedions! please do continue to share feedback with us - negative and positive! we're here to make it better for you and your community.

**Reply by triclambert** [reply]
+1 so excited to start using this in our own repo.
